element-hq/element-x-android
# Move migration screen to within the room list

## Type of change

- [x] Other : screen redesign.

## Content

- Moves the `migration` package contents from `:features:ftue` to `:features:messages:impl`.
- Adds light variant of the migration screen.
- Updates compound version to `v0.0.4` to fix status bar tinting issues.
- Displays the migration screen inside the room list one, allowing the user to send bug reports if needed, while preventing some other actions such as searching or creating a new room.

## Motivation and context

Closes #2310.

## Screenshots / GIFs

Inside the PR. There are also changes to the wait list screen since it reuses the same `SunsetPage` component, but I believe this is the desired behaviour.

Here's also a video with the current behaviour:

https://github.com/element-hq/element-x-android/assets/480955/29405a9f-66ca-46c0-86c8-3b7cf635a407

## Tests



- Remove all data from the EX app.
- Sign in again.
- Go through the FTUE screens quickly.
- With some luck, you'll still be able to see the migration screen and interact with the empty avatar icon now to open the settings.
- The migration screen should only appear once.

## Tested devices

- OS version(s): 14

## Checklist
